maintenance/wikia/RenameIP_local.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * @package MediaWiki
+ * @subpackage Maintanance
+ *
+ * @author: Władysław Bodzek <wladek@wikia-inc.com>
+ *
+ * @usage: SERVER_ID=177 php RenameIP_local.php --conf /usr/wikia/conf/current/wiki.factory/LocalSettings.php --aconf /usr/wikia/conf/current/AdminSettings.php
+ */
+
+ini_set( "include_path", dirname(__FILE__)."/../" );
+
+$options = array('help');
+
+$optionsWithArgs = array(
+	'old-ip',
+	'new-ip',
+	'old-ip-enc',
+	'new-ip-enc',
+	'phalanx-block-id',
+	'task-id',
+	'requestor-id',
+	'reason',
+	'global-task-id'
+);
+
+require_once( 'commandLine.inc' );
+global $IP, $wgCityId;
+
+if( isset( $options['help'] ) && $options['help'] ) {
+	echo( "Usage: SERVER_ID=target_cityId php RenameIP_local.php --rename-old-ip {string} --rename-new-ip {string} --requestor-id {int} [--phalanx-block-id {int}] [--task-id {int}] [--reason {string}] --conf {path} --aconf {path}\n\n" );
+	exit( 0 );
+}
+
+// BAC-602: decode user names if they were encoded in the first place
+if ( !empty( $options['old-ip-enc'] ) ) {
+	$options['old-ip'] = rawurldecode($options['old-ip-enc']);
+}
+if ( !empty( $options['new-ip-enc'] ) ) {
+	$options['new-ip'] = rawurldecode($options['new-ip-enc']);
+}
+
+if ( empty($options['old-ip']) || empty($options['new-ip']) ) {
+	echo( "Not enough arguments or invalid values. Required are: --old-ip, --new-ip");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the parentheses from this "echo" call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2041)

[project-id]: wikia:app